### PR TITLE
exec c01-e01 and c01-e03

### PR DESCRIPTION
**Pull request by Henriqueediniz**
# Description

Exercise (C01-E01 and C01-E03): 
Student Name: Henrique Diniz

i'm using command **aws --version** to show the Aws-cli version.
i'm using command **docker -v** to show the Docker-cli version.
i'm using command **terraform** version to show the Terraform-cli version.

You can see my execution in bash terminal in this route **/classes/01class/exercises/c01-e01**

_ps: Commit 72e2362 about c01-e02 has been reversed._

### **--  updating --** 

I commited a resolution about exercise **c01-e03**.

You can see the datails of how I created an instance with public IP and the instance with private IP in txt file.
Feactures:
- connect to it via SSH
- same VPC (default)
- I used the elastic IP to test my private IP connection.

## Type of change

- [x] Exercise Submission
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/01class/exercises/c01-e01/c01-e01.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+✦ 👽 henriquediniz aws --version
+aws-cli/1.17.5 Python/2.7.16 Darwin/19.2.0 botocore/1.14.5
+academy/classes/01class/Exercises/c01-e01 on  henriqueediniz-c01-e01 [?] 
+✦ 👽 henriquediniz docker -v
+Docker version 19.03.5, build 633a0ea
+academy/classes/01class/Exercises/c01-e01 on  henriqueediniz-c01-e01 [?] 
+✦ 👽 henriquediniz terraform version
+Terraform v0.12.19
+
+Your version of Terraform is out of date! The latest version
```
Comment (caiocezart) [inline classes/01class/exercises/c01-e01/c01-e01.txt]: Do you have a specific reason for not running the latest version? If no, can you update terraform and post the new version here?

Comment (Henriqueediniz) [reply classes/01class/exercises/c01-e01/c01-e01.txt]: No, I already update now.

**Comment by Henriqueediniz** [comment]
> Can you update the description of your pull request trying to explain what the written code is for. You just used the template without changing the values

I put more detail in description. Can you check?

File: classes/01class/exercises/c01-e01/c01-e01.txt (hunk 1 above)
Comment (caiocezart) [reply classes/01class/exercises/c01-e01/c01-e01.txt]: cool, can you update this file with the new version?

Comment (Henriqueediniz) [reply classes/01class/exercises/c01-e01/c01-e01.txt]: I updated now. You can check in this link https://github.com/devopsacademyau/academy/pull/21/commits/28a9d0648c37ed36f42808f7782c502911795693

**Comment by Henriqueediniz** [comment]
> C01-E02 doesn't need to be submitted. Can you remove it from your PR?
> Besides that, LGTM now

I'd committed with revert of changes about C01-E02

File: classes/01class/exercises/c01-e03/henriqueediniz/c01-e03.txt [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+[ec2-user@ip-172-31-85-181 ~]$ 
+
+
+
+Documentation reference: https://docs.aws.amazon.com/efs/latest/ug/gs-step-one-create-ec2-resources.html
+
+
+Private IP: 
+
+To access my other instance, I did not flag Assign IPs automatically as private and used the same configuration as the last instance, however, I have to create an elastic IP associated with my private IP to be accessed from within the VPC using the internal IP .
```
Comment (kikobr82) [inline classes/01class/exercises/c01-e03/henriqueediniz/c01-e03.txt]: Hey Henrique, thats good to see that you're going through the documentation to find how to solve the exercise.
In that case, the second instance can not not have any external IP, so an Elastic IP attached to it is not the way to go.
The way to connect to the second instance is through a `jump server` or `bastion host`. This means that you need to access one server that have a public ip(like the first server) and from that public accessible server, you connect to the second server that only have internal IP. That's the reason for the name `jump server`. You first need to get into the first server to "jump" into the second server.
Review that and update your c01-e03.txt file so we can approve, but i'ts great to see your dedication to this!

Comment (Henriqueediniz) [reply classes/01class/exercises/c01-e03/henriqueediniz/c01-e03.txt]: Hi Kiko, I understand what do you want say and made my exercise again. Can you check the answer?

**Pull request merged**